docker/copy-embedded-settings.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Let's call this `copy-embedded-settings` instead.

Reply:
Done